wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -94,11 +94,31 @@ default ParallelRaceGroup withTimeout(double seconds) {
    *
    * @param condition the interrupt condition
    * @return the command with the interrupt condition added
+   * @see #onlyWhile(BooleanSupplier)
    */
   default ParallelRaceGroup until(BooleanSupplier condition) {
     return raceWith(new WaitUntilCommand(condition));
   }
 
+  /**
+   * Decorates this command with a run condition. If the specified condition becomes false before
+   * the command finishes normally, the command will be interrupted and un-scheduled. Note that this
+   * only applies to the command returned by this method; the calling command is not itself changed.
```

Comment:
This is the same in all decorators, and the calling command object must not be used anymore. I don't see reason to specify that here more than the others do.

Reply:
I simply copied the doc comment from `until`, but I could remove that sentence from `until` and `onlyWhile`. On a related note, should I keep the longer paragraph note about compositions?

Reply:
That paragraph about not reusing composed instances should be there.

Reply:
I went ahead and removed the decorator sentence in `until` and `onlyWhile` (but left the note paragraph).